### PR TITLE
Also extract first workspace name from bindcodes

### PR DESCRIPTION
**Pull request by frsfnrrg**
This PR resolves an open TODO in `src/tree/workspace.c`, so that in cases like

         bindcode $mod+49 workspace 0
         bindcode $mod+10 workspace 1

the first workspace name is correctly identified.

### CODE REVIEW

File: sway/tree/workspace.c [hunk 1]
```diff
@@ -106,96 +106,104 @@ static bool workspace_valid_on_output(const char *output_name,
...
+		// Make sure that the command references an actual workspace
+		// not a command about workspaces
+		if (strcmp(_target, "next") == 0 ||
+			strcmp(_target, "prev") == 0 ||
+			strcmp(_target, "next_on_output") == 0 ||
+			strcmp(_target, "prev_on_output") == 0 ||
+			strcmp(_target, "number") == 0 ||
+			strcmp(_target, "back_and_forth") == 0 ||
+			strcmp(_target, "current") == 0)
+		{
```
Comment (SirCmpwn) [inline sway/tree/workspace.c]: style: put this brace on the previous line

**Comment by SirCmpwn** [comment]
LGTM aside from the one style isseu

File: sway/tree/workspace.c (hunk 1 above)
Comment (frsfnrrg) [reply sway/tree/workspace.c]: Done; also, indented by two tabs as per style and since the structure is otherwise confusing.

**Comment by RedSoxFan** [comment]
Thanks!

**Pull request merged**